### ISSUE TITLE
VS Code debug features randomly break with 1.57+

### ISSUE THREAD

**Issue by coryroloff** [microsoft/vscode#126704]
Issue Type: <b>Bug</b>

When running VS Code 1.57 (and now 1.57.1), the editor's Run and Debug panel is empty, the debug console is gone, and some plugins fail to render their UI. This appears to happen randomly each time I open the application (sometimes everything works fine, most of the time it doesn't). When I revert back to 1.56, I stop seeing these issues entirely.

I've tried manually disabling all extensions as well as launching with the --disable-extensions flag. Same behavior both ways.

I've checked VS Code's logs and while I don't know if these are related, here are a few errors I'm seeing. These occur both with all extensions enabled and all extensions disabled.

Log (Extension Host):
`[2021-06-18 13:17:51.927] [exthost] [error] [ExtHostStorage] {"code":"Unknown","name":"Error (FileSystemError)"}`

Log (Window):
`[2021-06-18 13:29:54.035] [renderer1] [error] Handler already set!: Error: Handler already set!`

VS Code version: Code 1.57.1 (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:32.912Z)
OS version: Darwin x64 20.5.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|12, 8, 8|
|Memory (System)|16.00GB (0.02GB free)|
|Process Argv|--crash-reporter-id 220c192d-db2d-4f14-96ac-d7c741028cda|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (39)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.2.0
cucumberautocomplete|ale|2.15.1
ng-template|Ang|12.0.5
swagger-viewer|Arj|3.0.1
vscode-toggle-quotes|Bri|0.3.4
transformer|dak|1.11.1
vscode-eslint|dba|2.1.23
EditorConfig|Edi|0.16.4
vscode-lombok|Gab|1.0.1
gc-excelviewer|Gra|3.0.42
todo-tree|Gru|0.0.213
vscode-test-explorer|hbe|2.20.4
vscode-auto-open-markdown-preview|hnw|0.0.4
vscode-todo-highlight|jgc|2.0.2
vscode-docker|ms-|1.13.0
vscode-kubernetes-tools|ms-|1.3.3
remote-containers|ms-|0.183.0
test-adapter-converter|ms-|0.0.10
debugger-for-chrome|msj|4.12.12
gradle-language|nac|0.2.3
vscode-concourse|Piv|1.25.1
vscode-spring-boot|Piv|1.25.1
angular-karma-test-explorer|raa|1.2.8
java|red|0.79.2
vscode-commons|red|0.0.6
vscode-xml|red|0.16.1
vscode-yaml|red|0.20.0
vscode-sort-json|ric|1.20.0
jinjahtml|sam|0.16.0
vscode-checkstyle|she|1.4.0
vscode-stylelint|sty|0.86.0
markdowntable|Tak|0.6.0
vscodeintellicode|Vis|1.2.14
vscode-java-debug|vsc|0.34.0
vscode-java-dependency|vsc|0.18.4
vscode-java-pack|vsc|0.16.0
vscode-java-test|vsc|0.30.0
vscode-icons|vsc|11.5.0
whitesource-advise|Whi|21.5.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255cf:30323309
vscorehov:30309549
vscod805:30301674
vscextlang:30322306
binariesv615cf:30323120

```

</details>

**Comment by weinand** [comment]
I was seeing this sporadically in Insiders too, but since last week it is gone.

**Comment by isidorn** [comment]
Thanks for filling this issue. Can you reproduce using Visual Studio Code Insiders https://code.visualstudio.com/insiders/

**Comment by coryroloff** [comment]
I was using the Insiders version all day today and I didn't see the issue even once, so it seems like 1.58 fixes the issue. Is it possible to push that fix to 1.57?

**Comment by isidorn** [comment]
No. We will soon ship a new release.
Also take into account that maybe one of your extensions is breaking this flow.
So for now closing this as no code change is planned
